FreewheelSampleApp/app/src/main/java/com/ooyala/sample/players/CustomConfiguredFreewheelPlayerActivity.java

```diff
@@ -102,6 +104,19 @@ public void update(Observable arg0, Object arg1) {
     if (arg1 == OoyalaPlayer.TIME_CHANGED_NOTIFICATION) {
       return;
     }
+
+
+
+    // Hook to write Notifications to a temporary file on the device/emulator
+    // Keeps track of incoming notifications and makes sure count is right
+    count++ ;
```

Comment:
Looking at http://github.services.ooyala.net/playback-app/android-sdk/pull/295, it now appears to me that the 'count' should be a private field of the SDCardLogcatOoyalaEventsLogger? Might be worth doing a code review in person to consider various options for this. Putting it into the class would help reduce the duplication that is being done here in all these sample apps you have to update as well, maybe. :-)